### DISCUSSION TITLE
Why dita-ot-action fails on Windows runner?

### DISCUSSION THREAD

**Discussion by syeda-git** [dita-ot/dita-ot#3936] [Q&A] [answered]
Hello,
I am setting up GitHub Actions on GHE server for dita builds. After crossing many hoops, i seem to be getting this error:
'Container action is only supported on Linux'
Am assuming it's because dita-ot-actions repo uses 'docker' for the build? If that's correct, how can i use a Windows runner in Actions to run a dita build?

The workflow:
name: CI
'on':
  push:
    branches:
      - master
jobs:
  build-dita:
    name: Build DITA
    runs-on: self-hosted
    steps:
      - name: Git checkout
        uses: UA-PNP/checkout-action@main
      - name: Build HTML5
        uses: UA-PNP/dita-ot-action@main
        with:
            build: |
                dita -i notes12/notes.ditamap -o out -f html5

**Comment by syeda-git** [comment]
@jason-fox Can you shed some light on this issue? I have cloned the dita-ot-action repo in my GitHub Enterprise Server. The runner is self-hosted on my Windows laptop. Are there equivalent Windows files for dita-ot-action?

**Comment by jason-fox** [comment, marked as answer]
The dita-ot-action is a Docker container Action. From: https://docs.github.com/en/actions/creating-actions/about-custom-actions

> Docker container actions can only execute on runners with a Linux operating system. Self-hosted runners must use 
> a Linux operating system and have Docker installed to run Docker container actions. For more information about the 
> requirements of self-hosted runners, see "[About self-hosted runners](https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners#requirements-for-self-hosted-runner-machines)."
